yb-voyager/src/pgss/parser.go

```diff
@@ -58,12 +58,13 @@ func ParseFromCSV(csvPath string) ([]PgStatStatements, error) {
 		}
 
 		// Entry is valid if parsing succeeded
-		entries = append(entries, *entry)
+		entries = append(entries, entry)
 		lineNumber++
 	}
 
 	log.Infof("PGSS CSV parsing completed with %d entries", len(entries))
-	return entries, nil
+	// since there is a possibility of same queryid for different entries(due to userid difference), we need to merge them
+	return MergePgStatStatements(entries), nil
```

Comment:
IMO, as much as possible, we should put all merge logic later in the compare-perf, not in the collection layer. (assessment)

Because what if for some other use-case of pg_stat_statements, we do not want a merge?

Okay to go ahead for now, but let's keep this in mind for later.